pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -142,7 +142,7 @@ public class OfflineClusterIntegrationTest extends BaseClusterIntegrationTestSet
   private static final String MAX_NUM_MULTI_VALUES_MAP_KEY = "maxNumMultiValuesMap";
   // TODO: This might lead to flaky test, as this disk size is not deterministic
   //       as it depends on the iteration order of a HashSet.
-  private static final int DISK_SIZE_IN_BYTES = 20796000;
+  private static final int DISK_SIZE_IN_BYTES = 20796348;
```

Comment:
why change on this?

Reply:
thanks @yupeng9 for review. 
I added "segment.use.ttl" properties in the metadata.
When load from disk, I use this properties from `segmentMetadataPropertiesConfiguration` to initialize [validDocSnapshot](https://github.com/apache/pinot/pull/9062/commits/6bfab718afadfaa7139de227dd27edf7d0a44b9e)